hack/bicep-types-radius/src/generator/src/cmd/generate.ts

```diff
@@ -256,7 +256,7 @@ async function buildTypeIndex(logger: ILogger, baseDir: string, version: string)
       types: readTypesJson(content),
     });
   }
-  const indexContent = await buildIndex(typeFiles,  log => logOut(logger, log), {name: "radius", version: version, isSingleton: false} as TypeSettings);
+  const indexContent = await buildIndex(typeFiles,  log => logOut(logger, log), {name: "radius", version: (version == "edge" ? "latest" : version), isSingleton: false} as TypeSettings);
```

Comment:
it would be nice if we could keep the logic about edge/latest out of this code and limit to the GH actions + makefile.

Reply:
Makes sense. Removed the check from here and added it to the build workflow instead